### PR TITLE
bug: Fixes cannot copy text content from sidebar

### LINKED ISSUE #3454: bug: Cannot copy text content from sidebar
With the new draggable sidebar feature there is no way one could copy text content from sidebar.

- We might need to disable the drag over content inside the content area
- Make only the title or collapsed box draggable, Show a different mouse pointer to tell user it's draggable `cursor: grab;`

https://user-images.githubusercontent.com/1277421/143174208-bc4c8f96-f0b1-4415-9096-ce4f00478b09.mov

@muhsin-k @iamsivin Any thoughts ?

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes #3454 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

**Screen recording** 

https://user-images.githubusercontent.com/64252451/144816409-ae645b17-0332-43ab-802f-15f0f7f1b6e0.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <draggable
       :list="conversationSidebarItems"
       :disabled="!dragEnabled"
+      :move="checkMove"
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue:12]:
```suggestion
      :move="handleMove"
```

File: app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue [hunk 2]
```diff
@@ -215,6 +216,12 @@ export default {
         conversation_sidebar_items_order: this.conversationSidebarItems,
       });
     },
+    checkMove(evt, event) {
+      if (event.target.className === 'cw-accordion--title') {
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue:220]:
```suggestion
      return event.target.className === 'cw-accordion--title');
```

**Comment by fayazara** [comment]
@sivin, we can solve this with drag handle prop

1. Add a handle selector to the draggable.

```
<draggable
      :list="conversationSidebarItems"
      :disabled="!dragEnabled"
      class="list-group"
      handle=".drag-handle"
      ghost-class="ghost"
      @start="dragging = true"
      @end="onDragEnd"
    >
``` 

Add this class to the button element in `AccordianItem.vue` file - 
```
<button class="cw-accordion--title drag-handle" @click="$emit('click')">
```

This will ensure, the items can be dragged as well as text can be copied from them


https://user-images.githubusercontent.com/15716057/144813340-ca6e7c55-5667-4214-9cff-b7373eee5675.mp4

**Comment by iamsivin** [comment]
Thanks @fayazara , I will change it.

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**